x-pack/plugins/observability_solution/entity_manager/server/lib/entities/find_entity_definition.ts

```diff
@@ -56,14 +53,14 @@ async function getEntityDefinitionState(
   esClient: ElasticsearchClient,
   definition: EntityDefinition
 ) {
-  const historyIngestPipelineId = generateHistoryIngestPipelineId(definition);
-  const latestIngestPipelineId = generateLatestIngestPipelineId(definition);
+  const historyIngestPipelineId = generateHistoryId(definition);
```

Comment:
i like that these are all aligned, but i find the concept of 'historyId' a little confusing to read.

i would prefer to keep the stub functions for `generateHistoryIngestPipelineId` etc, and have them all just call a common base function to return the same value.

Reply:
Sure thing!

Reply:
Fixed in https://github.com/elastic/kibana/pull/187507/commits/a4e68d7e2c705473eb5441be730a3f571c7c72a9